src/platforms/flutter/upload-debug.mdx

```diff
@@ -18,32 +18,53 @@ For Flutter Desktop (Windows/macOS/Linux) `split-debug-info` and `obfuscate` fla
+dev_dependencies:
+  sentry_dart_plugin: ^1.0.0
+```
+
+### Build
+
+The `flutter build apk`, `flutter build ios` (or _macos_) or `flutter build web` is required before executing the `sentry_dart_plugin` plugin, because the build spits out the debug symbols and source maps.
+
+### Run
+
+```bash
+flutter packages pub run sentry_dart_plugin
+```
+
+## Manually Upload Debug Symbols
+
+### Uploading for iOS and macOS
 
 Sentry requires a dSYM upload to symbolicate your crash logs. The symbolication process unscrambles Apple’s crash logs to reveal the function, file names, and line numbers of the crash. [Learn how to upload the dSYM files](/platforms/apple/dsym/).
 
 If you use the `split-debug-info` and `obfuscate` flags, you need to upload the `*.dSYM` files instead of the `*.symbols` files generated by the Flutter build. The `*.dSYM` files are located in the `build` folder of your project and not the given `split-debug-info` folder.
 
-## Uploading for Android NDK
+### Uploading for Android NDK
 
-If you are using `sentry_flutter` version earlier than 5.1, native symbolication on Android requires a specific configuration. For good symbolication, set `android:extractNativeLibs="true"` in your `AndroidManifest.xml` file or `android.bundle.enableUncompressedNativeLibs=false` for [Android App Bundle](https://developer.android.com/guide/app-bundle) in your `gradle.properties` file.
+The NDK symbols are uploaded by the Sentry Dart Plugin as well.
 
-If you use the [Android NDK](/platforms/android/using-ndk/), you must [upload files manually using our CLI](/product/cli/dif/#uploading-files).
+If you are using `sentry_flutter` version earlier than 5.1, native symbolication on Android requires a specific configuration. Refer to [Troubleshooting](/platforms/flutter/troubleshooting/) for more information.
 
 <Note>
 
 Sentry's Flutter SDK doesn't currently support the `uploadNativeSymbols` flag from the [Sentry Gradle Plugin](/platforms/android/gradle/).
 
 </Note>
 
-## Uploading with ProGuard
+### Uploading with ProGuard
 
-To use ProGuard with Sentry, upload the [Android Proguard/R8 mapping files](/platforms/android/proguard/) by either the recommended method of using our Gradle integration or manually by using [sentry-cli](/product/cli/dif/#proguard-mapping-upload).
+To use ProGuard with Sentry, upload the [Android Proguard/R8 mapping files](/platforms/android/proguard/) by either the recommended method of using our Gradle integration or manually by using [sentry-cli](/product/cli/dif/#proguard-mapping-upload). This is optional, and only needed if you have `proguard` enabled.
 
-## Uploading Source Maps for Flutter Web
+### Uploading Source Maps for Flutter Web
 
-You can also use our CLI to [upload source maps for Flutter Web](/product/cli/releases/#managing-release-artifacts).
+The Sentry Dart plugin uploads source maps as well. You can also use our CLI to [upload source maps for Flutter Web](/product/cli/releases/#managing-release-artifacts).
```

Comment:
Similar to https://github.com/getsentry/sentry-docs/pull/7097/files#r1219455786
with the `upload_source_maps` flag.